schema/data/test/mock_current_timestamp.sql

```diff
@@ -6,7 +6,7 @@ create or replace function ggircs_portal.current_timestamp() returns timestamptz
 $$
   select application_open_time
   from ggircs_portal.reporting_year
-  where reporting_year = 2018
+  where reporting_year = 2019
```

Comment:
@matthieu-foucault @dleard : Any clues as to why updating this would cause the `ciip_carbon_tax_calculation_test`  to fail?

Reply:
adding this to the top of the ciip_carbon_tax_calculation_test (under select plan) will fix the test by having it use the 2018 date.
`create or replace function ggircs_portal.current_timestamp() returns timestamptz as
$$
  select application_open_time
  from ggircs_portal.reporting_year
  where reporting_year = 2018
$$ language sql;`